### PR TITLE
chore: add maintainers as codeowners to all files

### LINKED ISSUE #10: Implement eth_call
Issues in epic:
- [x] https://github.com/hashgraph/hedera-hashio/issues/11
- [x] https://github.com/hashgraph/hedera-hashio/issues/12

### PR DESCRIPTION
**Pull request by PavelSBorisov**
**Description**:
Update CODEOWNERS to provide maintainers ownership of all files in their repos

**Related issue(s)**:

Fixes #10 

**Notes for reviewer**:
N/A

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -2,21 +2,21 @@
...
-*                                               @hashgraph/hedera-smart-contracts-product
+*                                               @hashgraph/devops-ci @hashgraph/release-engineering-managers @hashgraph/hedera-smart-contracts
 
 #########################
 ##### Hedera Relay ######
 #########################
-/packages/relay/                                @AlfredoG87 @ebadiere @lukelee-sl @hashgraph/hedera-smart-contracts-product
-/packages/server/                               @AlfredoG87 @ebadiere @Nana-EC @hashgraph/hedera-smart-contracts-product
-/packages/ws-server/                            @AlfredoG87 @ebadiere @georgi-l95 @Ivo-Yankov @hashgraph/hedera-smart-contracts-product
+/packages/relay/                                @AlfredoG87 @lukelee-sl @hashgraph/devops-ci @hashgraph/release-engineering-managers @hashgraph/hedera-smart-contracts
```
Comment (andrewb1269hg) [inline .github/CODEOWNERS:10]: Alfredo and luke are both in hedera-smart-contracts, so remove the individual user callouts.

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -2,21 +2,21 @@
...
+/packages/server/                               @AlfredoG87 @Nana-EC @hashgraph/devops-ci @hashgraph/release-engineering-managers @hashgraph/hedera-smart-contracts
```
Comment (andrewb1269hg) [inline .github/CODEOWNERS:11]: Alfredo and luke are both in hedera-smart-contracts, so remove the individual user callouts.

File: .github/CODEOWNERS [hunk 3]
```diff
@@ -2,21 +2,21 @@
...
+/packages/ws-server/                            @AlfredoG87 @georgi-l95 @Ivo-Yankov @hashgraph/devops-ci @hashgraph/release-engineering-managers @hashgraph/hedera-smart-contracts
```
Comment (andrewb1269hg) [inline .github/CODEOWNERS:12]: Alfredo is in the hedera-smart-contracts group, so remove the individual user callout.

File: .github/CODEOWNERS [hunk 4]
```diff
@@ -2,21 +2,21 @@
...
 
 ###############################
 ##### Tools and Examples ######
 ###############################
-/dapp-example/                                  @georgi-l95 @Ivo-Yankov @hashgraph/hedera-smart-contracts-product
-/k6/                                            @AlfredoG87 @hashgraph/hedera-smart-contracts-product
-/tools/                                         @georgi-l95 @Ivo-Yankov @hashgraph/hedera-smart-contracts-product
+/dapp-example/                                  @georgi-l95 @Ivo-Yankov @hashgraph/devops-ci @hashgraph/release-engineering-managers @hashgraph/hedera-smart-contracts
+/k6/                                            @AlfredoG87 @hashgraph/devops-ci @hashgraph/release-engineering-managers @hashgraph/hedera-smart-contracts
```
Comment (andrewb1269hg) [inline .github/CODEOWNERS:18]: Alfredo is in the hedera-smart-contracts group, so remove the individual user callout.

File: .github/CODEOWNERS [hunk 5]
```diff
@@ -26,15 +26,15 @@
 
 # Protection Rules for Github Configuration Files and Actions Workflows
 /.github/                                       @hashgraph/devops-ci @hashgraph/devops-ci-committers @hashgraph/release-engineering @hashgraph/release-engineering-managers
-/.github/workflows/                             @hashgraph/devops-ci @hashgraph/devops-ci-committers @hashgraph/release-engineering @hashgraph/release-engineering-managers @AlfredoG87 @ebadiere @Nana-EC @hashgraph/hedera-smart-contracts-product
+/.github/workflows/                             @AlfredoG87 @Nana-EC @hashgraph/devops-ci @hashgraph/devops-ci-committers @hashgraph/release-engineering @hashgraph/release-engineering-managers @hashgraph/hedera-smart-contracts
```
Comment (andrewb1269hg) [inline .github/CODEOWNERS:29]: Alfredo is in the hedera-smart-contracts group, so remove the individual user callout.

File: .github/CODEOWNERS [hunk 6]
```diff
@@ -26,15 +26,15 @@
...
 
 # Self-protection for root CODEOWNERS files (this file should not exist and should definitely require approval)
-/CODEOWNERS                                     @hashgraph/release-engineering @hashgraph/release-engineering-managers
+/CODEOWNERS                                     @hashgraph/release-engineering-managers
 
 # Protect the repository root files
-/README.md                                      @hashgraph/release-engineering @hashgraph/release-engineering-managers @AlfredoG87 @ebadiere @Nana-EC @hashgraph/hedera-smart-contracts-product
-**/LICENSE                                      @hashgraph/release-engineering @hashgraph/release-engineering-managers
+/README.md                                      @AlfredoG87 @Nana-EC @hashgraph/release-engineering-managers @hashgraph/devops-ci @hashgraph/hedera-smart-contracts
```
Comment (andrewb1269hg) [inline .github/CODEOWNERS:35]: Alfredo is in the hedera-smart-contracts group, so remove the individual user callout.

File: .github/CODEOWNERS [hunk 7]
```diff
@@ -26,15 +26,15 @@
...
+**/LICENSE                                      @hashgraph/release-engineering-managers
 
 # Git Ignore definitions
-**/.gitignore                                   @hashgraph/release-engineering @hashgraph/release-engineering-managers @AlfredoG87 @ebadiere @Nana-EC @hashgraph/hedera-smart-contracts-product
-**/.gitignore.*                                 @hashgraph/release-engineering @hashgraph/release-engineering-managers @AlfredoG87 @ebadiere @Nana-EC @hashgraph/hedera-smart-contracts-product
+**/.gitignore                                   @AlfredoG87 @Nana-EC @hashgraph/release-engineering @hashgraph/release-engineering-managers @hashgraph/devops-ci @hashgraph/hedera-smart-contracts
```
Comment (andrewb1269hg) [inline .github/CODEOWNERS:39]: Alfredo is in the hedera-smart-contracts group, so remove the individual user callout.

File: .github/CODEOWNERS [hunk 8]
```diff
@@ -26,15 +26,15 @@
...
+**/.gitignore.*                                 @AlfredoG87 @Nana-EC @hashgraph/release-engineering @hashgraph/release-engineering-managers @hashgraph/devops-ci @hashgraph/hedera-smart-contracts
```
Comment (andrewb1269hg) [inline .github/CODEOWNERS:40]: Alfredo is in the hedera-smart-contracts group, so remove the individual user callout.

**Review by andrewb1269hg — CHANGES_REQUESTED** [review]

**Review by andrewb1269hg — APPROVED** [review]

File: .github/CODEOWNERS [hunk 9]
```diff
@@ -2,21 +2,21 @@
 ##### Global Protection Rule ######
 ###################################
 # NOTE: This rule is overriden by the more specific rules below. This is the catch-all rule for all files not covered by the more specific rules below
-*                                               @hashgraph/hedera-smart-contracts-product
+*                                               @hashgraph/devops-ci @hashgraph/release-engineering-managers @hashgraph/hedera-smart-contracts
```
Comment (rbarkerSL) [inline .github/CODEOWNERS:5]:
```suggestion
*                                               @hashgraph/hedera-smart-contracts @hashgraph/hedera-smart-contracts-product 
```

File: .github/CODEOWNERS [hunk 10]
```diff
@@ -2,21 +2,21 @@
...
 
 #########################
 ##### Hedera Relay ######
 #########################
-/packages/relay/                                @AlfredoG87 @ebadiere @lukelee-sl @hashgraph/hedera-smart-contracts-product
-/packages/server/                               @AlfredoG87 @ebadiere @Nana-EC @hashgraph/hedera-smart-contracts-product
-/packages/ws-server/                            @AlfredoG87 @ebadiere @georgi-l95 @Ivo-Yankov @hashgraph/hedera-smart-contracts-product
+/packages/relay/                                @hashgraph/devops-ci @hashgraph/release-engineering-managers @hashgraph/hedera-smart-contracts
+/packages/server/                               @Nana-EC @hashgraph/devops-ci @hashgraph/release-engineering-managers @hashgraph/hedera-smart-contracts
+/packages/ws-server/                            @georgi-l95 @Ivo-Yankov @hashgraph/devops-ci @hashgraph/release-engineering-managers @hashgraph/hedera-smart-contracts
```
Comment (rbarkerSL) [inline .github/CODEOWNERS:12]:
```suggestion
/packages/relay/                                @hashgraph/hedera-smart-contracts @hashgraph/hedera-smart-contracts-product
/packages/server/                               @Nana-EC @hashgraph/hedera-smart-contracts @hashgraph/hedera-smart-contracts-product
/packages/ws-server/                            @georgi-l95 @Ivo-Yankov @hashgraph/hedera-smart-contracts @hashgraph/hedera-smart-contracts-product
```

File: .github/CODEOWNERS [hunk 11]
```diff
@@ -2,21 +2,21 @@
...
 
 ###############################
 ##### Tools and Examples ######
 ###############################
-/dapp-example/                                  @georgi-l95 @Ivo-Yankov @hashgraph/hedera-smart-contracts-product
-/k6/                                            @AlfredoG87 @hashgraph/hedera-smart-contracts-product
-/tools/                                         @georgi-l95 @Ivo-Yankov @hashgraph/hedera-smart-contracts-product
+/dapp-example/                                  @georgi-l95 @Ivo-Yankov @hashgraph/devops-ci @hashgraph/release-engineering-managers @hashgraph/hedera-smart-contracts
+/k6/                                            @hashgraph/devops-ci @hashgraph/release-engineering-managers @hashgraph/hedera-smart-contracts
+/tools/                                         @georgi-l95 @Ivo-Yankov @hashgraph/devops-ci @hashgraph/release-engineering-managers @hashgraph/hedera-smart-contracts
```
Comment (rbarkerSL) [inline .github/CODEOWNERS:19]:
```suggestion
/dapp-example/                                  @georgi-l95 @Ivo-Yankov @hashgraph/hedera-smart-contracts @hashgraph/hedera-smart-contracts-product
/k6/                                            @hashgraph/hedera-smart-contracts @hashgraph/hedera-smart-contracts-product
/tools/                                         @georgi-l95 @Ivo-Yankov @hashgraph/hedera-smart-contracts @hashgraph/hedera-smart-contracts-product
```

File: .github/CODEOWNERS [hunk 12]
```diff
@@ -26,15 +26,15 @@
 
 # Protection Rules for Github Configuration Files and Actions Workflows
 /.github/                                       @hashgraph/devops-ci @hashgraph/devops-ci-committers @hashgraph/release-engineering @hashgraph/release-engineering-managers
-/.github/workflows/                             @hashgraph/devops-ci @hashgraph/devops-ci-committers @hashgraph/release-engineering @hashgraph/release-engineering-managers @AlfredoG87 @ebadiere @Nana-EC @hashgraph/hedera-smart-contracts-product
+/.github/workflows/                             @Nana-EC @hashgraph/devops-ci @hashgraph/devops-ci-committers @hashgraph/release-engineering @hashgraph/release-engineering-managers @hashgraph/hedera-smart-contracts
```
Comment (rbarkerSL) [inline .github/CODEOWNERS:29]:
```suggestion
/.github/workflows/                             @hashgraph/devops-ci @hashgraph/devops-ci-committers @hashgraph/release-engineering @hashgraph/release-engineering-managers @hashgraph/hedera-smart-contracts @hashgraph/hedera-smart-contracts-product
```

File: .github/CODEOWNERS [hunk 13]
```diff
@@ -26,15 +26,15 @@
...
 
 # Self-protection for root CODEOWNERS files (this file should not exist and should definitely require approval)
-/CODEOWNERS                                     @hashgraph/release-engineering @hashgraph/release-engineering-managers
+/CODEOWNERS                                     @hashgraph/release-engineering-managers
 
 # Protect the repository root files
-/README.md                                      @hashgraph/release-engineering @hashgraph/release-engineering-managers @AlfredoG87 @ebadiere @Nana-EC @hashgraph/hedera-smart-contracts-product
-**/LICENSE                                      @hashgraph/release-engineering @hashgraph/release-engineering-managers
+/README.md                                      @Nana-EC @hashgraph/release-engineering-managers @hashgraph/devops-ci @hashgraph/hedera-smart-contracts
+**/LICENSE                                      @hashgraph/release-engineering-managers
```
Comment (rbarkerSL) [inline .github/CODEOWNERS:36]:
```suggestion
/README.md                                      @Nana-EC @hashgraph/release-engineering-managers @hashgraph/devops-ci @hashgraph/hedera-smart-contracts @hashgraph/hedera-smart-contracts-product 
**/LICENSE                                      @hashgraph/release-engineering-managers @hashgraph/hedera-smart-contracts-managers
```

File: .github/CODEOWNERS [hunk 14]
```diff
@@ -26,15 +26,15 @@
...
 
 # Git Ignore definitions
-**/.gitignore                                   @hashgraph/release-engineering @hashgraph/release-engineering-managers @AlfredoG87 @ebadiere @Nana-EC @hashgraph/hedera-smart-contracts-product
-**/.gitignore.*                                 @hashgraph/release-engineering @hashgraph/release-engineering-managers @AlfredoG87 @ebadiere @Nana-EC @hashgraph/hedera-smart-contracts-product
+**/.gitignore                                   @Nana-EC @hashgraph/release-engineering @hashgraph/release-engineering-managers @hashgraph/devops-ci @hashgraph/hedera-smart-contracts
+**/.gitignore.*                                 @Nana-EC @hashgraph/release-engineering @hashgraph/release-engineering-managers @hashgraph/devops-ci @hashgraph/hedera-smart-contracts
```
Comment (rbarkerSL) [inline .github/CODEOWNERS:40]:
```suggestion
**/.gitignore                                   @Nana-EC @hashgraph/release-engineering @hashgraph/release-engineering-managers @hashgraph/devops-ci @hashgraph/hedera-smart-contracts @hashgraph/hedera-smart-contracts-product 
**/.gitignore.*                                 @Nana-EC @hashgraph/release-engineering @hashgraph/release-engineering-managers @hashgraph/devops-ci @hashgraph/hedera-smart-contracts @hashgraph/hedera-smart-contracts-product 
```

**Review by rbarkerSL — CHANGES_REQUESTED** [review]

File: .github/CODEOWNERS [hunk 15]
```diff
@@ -26,15 +26,15 @@
...
 
 # Self-protection for root CODEOWNERS files (this file should not exist and should definitely require approval)
-/CODEOWNERS                                     @hashgraph/release-engineering @hashgraph/release-engineering-managers
+/CODEOWNERS                                     @hashgraph/release-engineering-managers
 
 # Protect the repository root files
-/README.md                                      @hashgraph/release-engineering @hashgraph/release-engineering-managers @AlfredoG87 @ebadiere @Nana-EC @hashgraph/hedera-smart-contracts-product
-**/LICENSE                                      @hashgraph/release-engineering @hashgraph/release-engineering-managers
+/README.md                                      @Nana-EC @hashgraph/release-engineering-managers @hashgraph/devops-ci @hashgraph/hedera-smart-contracts @hashgraph/hedera-smart-contracts-product 
+**/LICENSE                                      @hashgraph/release-engineering-managers @hashgraph/hedera-smart-contracts-managers
```
Comment (PavelSBorisov) [inline .github/CODEOWNERS:36]: @rbarkerSL the @hashgraph/hedera-smart-contracts-managers team doesn't seem to be valid here according to GitHub

Comment (andrewb1269hg) [reply .github/CODEOWNERS:36]: I added the team to the repository with write access. This should resolve the error we're seeing.

**Review by andrewb1269hg — APPROVED** [review]

**Review by quiet-node — APPROVED** [review]

**Review by rbarkerSL — APPROVED** [review]

**Pull request merged**